tomcat/add-classpath-entries.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+mkdir -p "{{.libDir}}"
+while IFS=: read -d: -r path; do
+    printf "Linking $path to '{{.libDir}}/$(basename $path)'\n"
```

Comment:
There are going to be various [ShellCheck](https://www.shellcheck.net/) improvements to the script itself (I use the IntelliJ integration pretty heavily), but since I haven’t automated the integration, I can just polish it up on the merge.  However, if you want to have a crack at it, please feel free but without obligation.

Reply:
Good point. I changed it so that Shellcheck is satisfied. While doing so, I noticed that the classpath can potentially contain a * wildcard, which should expand to all '.jar' and '.JAR' files (see [here](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/classpath.html#A1100762)).  I'll see if I can solve this easily in the script, but at least I'll add a warning if the classpath contains '*'.

Reply:
That's such an exceedingly rare use-case that I wouldn't worry about it at this time.  We my get a request to add the feature, but I'm comfortable not supporting it today.

Reply:
Ok, great. Thanks for the review!:)